squad/frontend/static/squad/controllers/charts.js

```diff
@@ -231,16 +229,50 @@ function ChartPanel($http, DATA) {
                 })
             })
 
-            var datasets = _.map(selected_environments, function(env) {
-                return {
+            var datasets = []
+            _.each(selected_environments, function(env) {
+                var line = chartPanel.filterData(data[env.name], minLimit, maxLimit)
+                var low = chartPanel.filterData(data[env.name], minLimit, maxLimit, 6)
+                var high = chartPanel.filterData(data[env.name], minLimit, maxLimit, 7)
+
+                // lower end of the range
+                if (low != line) {
+                    datasets.push({
+                        label: ':low:' + env.name,
+                        data: low,
+                        backgroundColor: env.range_color,
+                        pointRadius: 0,
+                        pointHoverRadius: 0,
+                        borderColor: '#ffffff00', // transparent
+                        yidx: 6,
+                        fill: '+1'
+                    }
+                    )
```

Comment:
i think this parenthesis should've been in line above, right after closing curly braces

Reply:
fixed